frontend/src/metabase/components/form/StandardForm.jsx

```diff
@@ -51,6 +51,21 @@ const StandardForm = ({
     </div>
     <div className="flex">
       <div className="ml-auto flex align-center">
+        {submitResultDetail && !dirty && (
+          <FormMessage
+            formError={
+              submitResultDetail && !submitResultDetail.success
+                ? submitResultDetail
+                : undefined
+            }
+            formSuccess={
+              submitResultDetail && submitResultDetail.success
+                ? submitResultDetail
+                : undefined
+            }
```

Comment:
It might be easier to read if you duplicate `<FormMessage>`?

```js
submitResultDetail.success ? (
  <FormMessage formSuccess={submitResultDetail} />
) : (
  <FormMessage formError={submitResultDetail} />
)
```

Reply:
This could be further simplified if we replaced `submitResultDetail ` with the `formError` and `formSuccess` props mentioned above:

```js
{(formError || formSuccess) && !dirty && (
  <FormMessage formError={formError} formSuccess={formSuccess} />
}
```